core-bundle/contao/themes/flexible/styles/styles/fieldset.pcss

```
@@ -51,3 +51,10 @@ fieldset.collapsed > legend > button,
 fieldset.collapsed > legend:not(:has(> button)) {
     background-image: url("icons/navexp.svg");
 }
+
+@media (min-width: 768px) {
+    fieldset:is(.tl_tbox,.tl_box) > legend > button {
```

```suggestion
    fieldset:is(.tl_tbox,.tl_box) > legend {
        padding-left: 43px;
        background-position-x: 23px;
    }

    fieldset:is(.tl_tbox,.tl_box) > legend:has(> button) {
        padding: 0;
    }

    fieldset:is(.tl_tbox,.tl_box) > legend > button {
```

Checked it with another extension and this is also needed, everything else however lgtm

---

Resolved in f25a5440bacafbfe1e62f26bdb4a4efc4398571e